### DISCUSSION TITLE
Resolve equation

### DISCUSSION THREAD

**Discussion by Clems02** [josdejong/mathjs#3245] [Q&A]
Hello everyone,
I am trying to use the library to create an application to solve an equation.
The equation can take simple forms like ax+b = 0 or more "complex" forms like ab+c(dx+e) = 0
Before solving the equation, you have to expand it to find a form ax + b.
However, I am stuck at this stage and cannot find a solution.

I tried to use the simplify function but it's not work. The equation is not expanded.
Do you have any clue please?

Thanks

My code:
```js
(equation entry:   3*(-7+x) = 0 )

export const resolveEquation = (equa: string) => {
  const equation = emptySpace(equa);

  const splitedPart = equation.split("=");

  let leftPart = splitedPart[0];
  let rightPart = splitedPart[1];

  let fullEquation = `${leftPart}-(${rightPart})`;

  let simplifiedEquation = simplify(fullEquation).toString();
};

output with log:   3 * (x - 7)
```

**Comment by josdejong** [comment]
Solving an equation involves turning the expression "inside out", extracting the variable out of it. It's not something that `simplify` can do for you. 

Related: https://github.com/josdejong/mathjs/discussions/2662
